kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/cloud/Cloudlet.java

```diff
@@ -79,7 +78,7 @@ public void unsetCloudService(CloudService cloudService) {
     //
     // ----------------------------------------------------------------
 
-    protected void activate(ComponentContext componentContext) {
```

Comment:
Yes. Because the `Cloudlet` API imposes restrictions/requirements on its implementors which are actually unnecessary. So if there is no requirement in the Cloudlet, then it should not require this. So it will be way easier for unit tests to actually test this.